xml/admin_gui_dashboard.xml

```diff
@@ -2676,12 +2682,12 @@ Please consult the documentation on how to configure and enable the Object Gatew
    <title>Enabling the &ogw; Management Front-end</title>
    <para>
     To use the &ogw; management functionality of the dashboard, you need to
-    provide the login credentials of a user with the 'system' flag enabled:
+    provide the login credentials of a user with the <option>system</option> flag enabled:
```

Comment:
This and the following change don't seem to be related to this PR?

Reply:
Yeah that's just me adding in syntax :eyes:

Reply:
"when in Rome"